### PR TITLE
Feature/ci improvement

### PR DESCRIPTION
**Pull request by Evoluty**
Fix issue #15 by adding php versions for travis and sonar cloud linter

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
+sonar.host.url=https://sonarcloud.io
+sonar.organization=evoluty-github
+sonar.projectKey=Evoluty_google-maps-apis-php-client
+sonar.projectName=google-maps-client
+sonar.projectVersion=1.0
+sonar.sources=.
+sonar.sourceEncoding=UTF-8
+sonar.exclusions=vendor/**,sonar-scanner*/**,tests/**,reports/**
+sonar.tests=tests
+sonar.externalIssuesReportPaths=reports/sonar-codesniffer-results.json
```
Comment (juherr) [inline sonar-project.properties]: >  sonar.externalIssuesReportPaths=reports/sonar-codesniffer-results.json
won't work in your case

File: sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,12 @@
+sonar.host.url=https://sonarcloud.io
+sonar.organization=evoluty-github
+sonar.projectKey=Evoluty_google-maps-apis-php-client
+sonar.projectName=google-maps-client
+sonar.projectVersion=1.0
+sonar.sources=.
+sonar.sourceEncoding=UTF-8
+sonar.exclusions=vendor/**,sonar-scanner*/**,tests/**,reports/**
```
Comment (juherr) [inline sonar-project.properties]: >  sonar.exclusions=vendor/**,reports/**

should be enough with 

> sonar.sources=src

File: sonar-project.properties [hunk 3]
```diff
@@ -0,0 +1,12 @@
+sonar.host.url=https://sonarcloud.io
```
Comment (juherr) [inline sonar-project.properties]: Remove this line, the travis plugin already set it

File: .travis.yml [hunk 4]
```diff
@@ -7,7 +7,12 @@ install:
    - travis_retry composer install --no-interaction --no-suggest
 
 script:
-  - composer test -- --coverage-clover=coverage.xml
+  - composer test -- --log-junit=phpunit.junit.xml --coverage-clover=coverage.xml
+  - sonar-scanner
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+
```
Comment (juherr) [inline .travis.yml:18]: Add somewhere
```
cache:
  directories:
- '$HOME/.sonar/cache'
```

Comment (Evoluty) [reply .travis.yml:18]: what's the point of those lines ?

**Review by juherr — APPROVED** [review]

File: .travis.yml (hunk 4 above)
Comment (Evoluty) [reply .travis.yml:18]: I'll skip those atm and add them later if needed

**Pull request merged**